### ISSUE TITLE
Bug: tracer tries to compile output of expression

### ISSUE THREAD

**Issue by kondziu** [PRL-PRG/R-dyntrace#105]
This is weird:
```
> trace.promises.db(print("hello world"))
Error: could not execute prepared statement "\[1] "hello world"
Elapsed time: 0.0301144123077393
[1] TRUE
```
It has to be first trace. It doesn't manifest for uncompiled.sql.

**Comment by kondziu** [comment]
Appeared recently, I think. I don't get that on 813b635a5068ee33cfcfce5684e00ddf5cf3cf29

**Comment by kondziu** [comment]
2854768474b4bde9633b3a095f2ea4eb3b23dd76 is the first bad commit
commit 2854768474b4bde9633b3a095f2ea4eb3b23dd76
Author: Konrad Siek <konrad.siek@cs.put.edu.pl>
Date:   Mon May 1 16:36:02 2017 -0400

    R interface for Sqlite stuff (using dplyr).
    
    Fixes #102

:040000 040000 41b2ba778681f3bd1fd95269e3fdedaa20c7b1de 88eb6de8d91d5ccc3f0f0e7d2315c1335e4d7c33 M      rdt-plugins
